docs/release-notes/index.md

```diff
@@ -1,5 +1,17 @@
 # Release notes
 
+## Jan 30, 2024 - CHILI GraFx
+
+![rn_icon](https://chilipublishdocs.imgix.net/logos/CHILI_LOGOS_OK-04.svg)
+
+### Features
+
+- Introducing User Groups to invite users in bulk to environments with a specific role per environment
+- A user can be added to a user group when they are invited to CHILI GraFx for the first time or from the User Management page on CHILI GraFx
+- User Group membership, User Group details (name & description) and assignement of environment and roles are managed by the Subscription Admin
+
+[Documentation on user groups](/CHILI-GraFx/guides/manage-user-groups/)
+
```

Comment:
### Features

- You can move an asset from one place within the environment to another using the quick actions menu.